src/Evolution/Systems/GrMhd/ValenciaDivClean/KastaunEtAlHydro.tpp

```diff
@@ -39,9 +39,8 @@ class FunctionOfZ {
               const double electron_fraction, const EosType& equation_of_state,
               const double lorentz_max)
       : q_(tau / rest_mass_density_times_lorentz_factor),
-        r_squared_(momentum_density_squared /
-                   square(rest_mass_density_times_lorentz_factor)),
-        r_(std::sqrt(r_squared_)),
+        r_(std::sqrt(momentum_density_squared /
```

Comment:
why sqrt a square?

Reply:
I agree, it shouldn't be needed, but might help with robustness against negative evolved vars, so I would prefer to keep it to match the previous algorithm...